crates.json

```diff
@@ -65,6 +65,11 @@
       "build": "cargo build --examples --all-features",
       "target": "thumbv7em-none-eabihf"
     },
+    "matrix_portal_m4": {
+      "tier": 1,
```

Comment:
This board will be a Tier 2 rather than 1.

Reply:
Changed `"tier": 1` to `"tier": 2`